test/integration/dataplane_scale_subresource_test.go

```diff
@@ -17,6 +22,18 @@ import (
 	operatorv1beta1 "github.com/kong/kubernetes-configuration/api/gateway-operator/v1beta1"
 )
 
+// run_kubectl_scale runs the kubectl scale command to test the scale subresource for the DataPlane CRD.
+// It scales the DataPlane resource to the specified number of replicas and returns the output and error.
+func run_kubectl_scale(t *testing.T, namespace, name string, replicas int) (string, error) {
```

Comment:
```suggestion
// runKubectlScale runs the kubectl scale command to test the scale subresource for the DataPlane CRD.
// It scales the DataPlane resource to the specified number of replicas and returns the output and error.
func runKubectlScaleDataPlane(t *testing.T, namespace, name string, replicas int) (string, error) {
```

Also: WDYT about using `types.NamespacedName` to prevent accidental errors from providing name first instead of the namespace?